code/modules/mob/living/carbon/xenomorph/abilities/praetorian/praetorian_powers.dm

```diff
@@ -317,12 +350,23 @@
 
 
 		shake_camera(H, 10, 1)
+
+		var/obj/effect/beam/tail_beam = X.beam(H, "oppressor_tail", 'icons/effects/beam.dmi', 0.5 SECONDS, 8)
+		var/image/tail_image = image('icons/effects/status_effects.dmi', "hooked")
+		H.overlays += tail_image
+
 		H.throw_atom(throw_target_turf, get_dist(throw_target_turf, H)-1, SPEED_VERY_FAST)
 
+		qdel(tail_beam) // hook beam catches target, throws them back, is deleted (throw_atom has sleeps), then hook beam catches another target, repeat
+		addtimer((CALLBACK(src, .proc/remove_tail_overlay, H, tail_image)), 0.5 SECONDS) //needed so it can actually be seen as it gets deleted too quickly otherwise.
```

Comment:
do try to avoid .proc/ as they will be deleted in 515

Reply:
fixed